crates/ide-db/src/syntax_helpers/format_string.rs

```diff
@@ -1,9 +1,11 @@
 //! Tools to work with format string literals for the `format_args!` family of macros.
 use syntax::{
     ast::{self, IsString},
-    AstNode, AstToken, TextRange, TextSize,
+    TextRange, TextSize,
 };
 
+use super::node_ext::get_outer_macro;
```

Comment:
```suggestion
use crate::syntax_helpers::node_ext::get_outer_macro;
```
We generally prefer not to use `super` paths (exception usually are test modules)

Reply:
done